ansible/roles/provision-files/vars/main.yml

```diff
@@ -122,15 +122,15 @@ girder_file_downloads:
       sha512: 7183385f8eaca85997725a107a76034de2bd4a59c1434b4bdb7c1ac8931cf4b68a53f6e736734643386364b9f0856de795a14965b6a02bc5eb5891252e6a73c9
       dest: "{{ stage_dirs.object_detector }}/r18_det.pt"
     # Activity classifier
-    - file_id: 663e7294687336214e7cdc07
-      sha512: b0166110745c39bb14447ddde34f789fcf7808dc8443a5dc1371de484e995e4fc24b426d1b884321ac51f63c88560ccbbed3a09be5006afa69445ec8d9e04151
+    - file_id: 66463a80687336214e7cdeaa
+      sha512: 11c417480ef3b178cea5e9ba01878e2c1d2978129db4d8ac5d7eb0a1e6bdec0da74fc54f6bd69c8d04311809fc78503fad5c57fe45c5ee09f0f4f68197e34be1
       dest: "{{ stage_dirs.activity_classifier }}/r18_tcn.ckpt"
     - file_id: 6606b6e9aa5c8de3874c3f4a
       sha512: 3c84333390ee6b12327bb7e5debed37149c7f95cc437b16939f77b599d1a0b3b8c4f0995820b95973170b94df695494000a1c45fbf46632267c212e125fe58a3
       dest: "{{ stage_dirs.activity_classifier }}/r18_mapping.txt"
     # Global Step predictor model
-    - file_id: 663e4c5d687336214e7cdbff
-      sha512: 12b5bf950f64d9609182f795f7243edb6438b3919d920f0f527127660bb632abd5917ed149c28aa5ebcef7e7f6d8ed6edff39b27286eb0ce37a1ff7de2da4c77
+    - file_id: 66464bf9687336214e7cdeae
+      sha512: bc7884c258cb8704372dd69a3e7d999eaf70d716d271982077c7216ef27ab01219ef1e488f6f9669e11a7e8aa6ffb9d8e07d74edc47294f90cc9e6a723f5a822
```

Comment:
Double check which models this refers to?

Reply:
Added a "Girder item ID" field to the training spreadsheet that will match the `file_id` here. The GSP will always be trained on the TCN model that is being downloaded (I'd consider it an error if it didn't)